### PR TITLE
Fix the tl;dr in the README.

### PR DESCRIPTION
**Pull request by mbrookes**
Pedantic bit twiddling:

1) tl;dr is usually separated by a semicolon, not a comma, as this (semi ironically) is more grammatically correct. 
2) No space between the clauses.
3) No need for the exclamation mark - people who know this meme know it's a joke. (And those who don't can now search with a better chance of finding it).
4) Lower case looks wrong for a title, so making it upper case which is an accepted variation.

tl;dr: I fixed your tl;dr. :laughing:

### CODE REVIEW

**Comment by martinhbramwell** [comment]
This is good.  Thanks.  

So, apart from that -- does the Kickstarter work for you? How are you using it?

**Comment by mbrookes** [comment]
I haven't yet, but I have an affection for Meteor and drop by the forum occasionally so when I saw this I had to check it out. Looks pretty comprehensive!
